history/taskana-simplehistory-rest-spring-example/src/main/java/pro/taskana/simplehistory/rest/ExampleRestApplication.java

```diff
@@ -19,7 +19,7 @@
 
 /** Example Application showing the implementation of taskana-rest-spring. */
 @SpringBootApplication
-@ComponentScan(basePackages = "pro.taskana.simplehistory.rest")
+@ComponentScan("pro.taskana.simplehistory.rest")
```

Comment:
We should be able to delete this ComponentScan annotation since this class already resides in the pro.taskana.simplehistory.rest package and the integrated @ComponentScan annotation (without arguments) in @SpringBootApplication should tell Spring to scan the current package and all of its sub-packages

Reply:
done